DependencyInjection/Configuration.php

```diff
@@ -34,8 +34,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('linkin_swagger_resolver');
+        $treeBuilder = new TreeBuilder('linkin_swagger_resolver');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
Please use this construct
```php
        if (method_exists($treeBuilder, 'getRootNode')) {
            $rootNode = $treeBuilder->getRootNode();
        } else {
            // BC layer for symfony/config 4.1 and older
            $rootNode = $treeBuilder->root('linkin_swagger_resolver');
        }
```
And do not forget import `method_exists` into `use` section